pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -377,7 +377,11 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateStatus(observedGeneration
 	}
 
 	cephFilesystemSubVolumeGroup.Status.Phase = status
-	cephFilesystemSubVolumeGroup.Status.Info = map[string]string{"clusterID": buildClusterID(cephFilesystemSubVolumeGroup)}
+	cephFilesystemSubVolumeGroup.Status.Info = map[string]string{
+		"clusterID": buildClusterID(cephFilesystemSubVolumeGroup),
+		"pinning":   getPinning(cephFilesystemSubVolumeGroup),
```

Comment:
why are we adding pinning details to the status here?

Reply:
Yes correct this is k8s status it is not needed

Reply:
Yes, this can be skipped if we emit the pinning from the output.
I think why this would have a benefit is that it might not be clear for the user if they are applying a configuration without pinning it will default Distributed